README.md

```diff
@@ -55,13 +54,14 @@ Install it over the version of pythontex that was just installed.
 
 (Note that on arch-based systems, one should use tllocalmgr instead.)
 
+## Test
+
+`make test` pulls all the codes and test those in `juliatest` block. Check the details of `runtests.jl`.
```

Comment:
Running `make test` pulls all the code and then runs all tests in `juliatest` blocks. See `runtests.jl` for details.